CLAUDE.md

```
@@ -116,6 +116,20 @@ See `mobile/CLAUDE.md` for Flutter architecture, Provider state management, and
 - Mobile uses `flutter_secure_storage` for secure token storage
 - All API endpoints except `/api/auth/login` and `/api/auth/signup` require authentication
 
+### Authorization (Roles & Permissions)
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)